apps/app/src/components/AppTitle.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import Link from "next/link";
+import { useIsHome } from "../hooks/useIsHome";
+import { AppLogo } from "./AppLogo";
+
+type TitleTag = keyof Pick<JSX.IntrinsicElements, "h1" | "div">;
+
+export const AppTitle = () => {
+	const isHome = useIsHome();
+	const Tag: TitleTag = isHome ? "h1" : "div";
```

Comment:
Czy ten typ jest w ogóle potrzebny?

Reply:
Dobra rozumiem, skoro wartość zmiennej `Tag` nie jest pobierana np. z innego komponentu i przekazywana do `AppTitle` np. jako props to nie zapobiega to zbytnio pomyłkom, bo ... pomylić można się tylko w jednym miejscu 😁